modules/service/src/main/scala/ch/epfl/bluebrain/nexus/commons/service/persistence/CassandraStorageHelper.scala

```diff
@@ -0,0 +1,71 @@
+package ch.epfl.bluebrain.nexus.commons.service.persistence
+
+import akka.Done
+import akka.actor.ExtendedActorSystem
+import akka.event.Logging
+import akka.persistence.cassandra.CassandraPluginConfig
+import akka.persistence.cassandra.session.scaladsl.CassandraSession
+import com.datastax.driver.core.Session
+import com.typesafe.config.Config
+
+import scala.concurrent.{ExecutionContext, Future}
+
+object CassandraStorageHelper {
+
+  /**
+    * Initialize Cassandra session and create keyspace and table if not exists.
+    *
+    * @param name        the prefix for the config property where to find the table name
+    * @param tableSchema the schema definition when creating the table
+    * @param system the actor system
+    * @return a [[Tuple3]] with the session, the keyspace and the table name
+    */
+  def apply(name: String, tableSchema: String, system: ExtendedActorSystem): (CassandraSession, String, String) = {
```

Comment:
Could we make this protected, rename it to something like `createSession` and change the object into a trait to be mixed in by different implementations of `...CassandraStorage`? Each implementation can then ensure it has created with keyspace / tables and have a reference to the session.